Jenkinsfile

```diff
@@ -166,10 +166,12 @@ pipeline {
                 SONAR_TOKEN = credentials('sonarqube-token')
             }
             steps {
-                // -DskipITs is temporary to skip all the tests that were failing at the time. See https://github.com/codice/ddf/issues/5777
-                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
-                    script {
-                        sh 'mvn -q -B -DskipITs -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent sonar:sonar -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -Dsonar.exclusions=${COVERAGE_EXCLUSIONS} -pl !$DOCS,!$ITESTS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
```

Comment:
❔ Do you think it is worth a comment here on why we are adding this? Basically we are seeing the stage fail when trying to upload the analysis.

Reply:
❔ Wouldn't `null` be better here, if the build is marked unstable in an earlier stage catchError will fail because it can't set the buildResult higher than what it was before catchError.

Reply:
Hmm, didn't think about that. Good point. Wasn't sure if it would override a previous stage buildResult. That wouldn't be ideal either.

Reply:
made buildResult null and stageResult 'FAILURE'